gulpfile.js

```diff
@@ -26,16 +24,7 @@ gulp.task('eslint', function () {
     .pipe(eslint.failOnError());
 });
 
-gulp.task('gen-prefs', function(cb){
-  var contents = '// Set prefs to use a local content server\n'; // eslint-disable-line
-  contents += 'user_pref("services.universalSearch.frameURL", "https://localhost:8080/index.html");\n';
-  contents += 'user_pref("services.universalSearch.baseURL", "https://localhost:8080/");\n';
-  contents += '\n// Set prefs using remote content server\n';
-  contents += '//user_pref("services.universalSearch.frameURL", "https://d1fnkpeapwua2i.cloudfront.net/index.html");\n';
-  contents += '//user_pref("services.universalSearch.baseURL", "https://d1fnkpeapwua2i.cloudfront.net/");\n';
-  fs.writeFile('user.js', contents, cb);
-});
-
+// TODO: stop doing this templating for the rdf files. It's excessively complex.
```

Comment:
Sadly, I don't think we can use jpm, since we aren't using the sdk. File a bug if I'm wrong, though :-)